server/v2/timeframe.js

```diff
@@ -0,0 +1,159 @@
+const FRAME_REG = /^(?:last\s)?(?:(\d+|a)\s)?((?<!^)[a-z]+?)s?(?:\sago)?$/i;
+const DATE_REG = /^([a-z]+?)?(?:\s(\d{1,2})(?:st|nd|rd|th))?\s?(\d{4})?$/i;
+const ISO_REG = /^(\d{4})(?:-(\d{1,2})(?:-(\d{1,2})(?:T(\d{2})(?::(\d{2})(?::(\d{2}))?)?)?)?|-?W(\d{2})(\d)?)$/i;
+
+/* there is a lot of similar but different date suff here and these rules make
+ * it more verbose than it already is; disable them */
```

Comment:
I'm gonna be you and pick at some styling 😉 

```
/*
 * there is a lot of similar but different date stuff here and these rules make
 * it more verbose than it already is; disable them 
*/
```

"Suff" typo :p

Reply:
🙏 I hate when I discover my own typos